INSTALL.md

```diff
@@ -156,3 +156,75 @@ Example:
 ```
 Lumen (5.3.2) (Laravel Components 5.3.*)
 ```
+
+## Installing ThesauRex as part of [Spacialist](https://github.com/eScienceCenter/Spacialist)
+
+As ThesaurRex is a crutial part of the [Spacialist](https://github.com/eScienceCenter/Spacialist) platform, it can be installed directly within the Spacialist environment as well. This part only contains changes that must be made when installing ThesauRex as part of Spacialist against a standalone installation.
+
+2. `cd` into your Spacialist Repository and clone the ThesauRex Repository into it
+
+   ```bash
+   git clone https://github.com/eScienceCenter/ThesauRex thesaurex
+   ```
+
+### Proxy Setup
+
+2. Add one entry for your spacialist and one for thesaurex to your hosts file
+
+   ```bash
+    sudo nano /etc/hosts
+    # Add an entry to "redirect" a domain to your local machine (localhost)
+    127.0.0.1 project.spacialist # or anything you want
+    127.0.0.1 project.thesaurex # or anything you want
+   ```
+
+3. Add an additional configuration to the vHost of you apache that you created while installing Spacialist
+
+   ```bash
+    cd /etc/apache2/sites-available
+    sudo nano thesaurex-lumen.conf
+   ```
+
+    Paste the following snippet into the file:
```

Comment:
The first `<VirtualHost >` is the Spacialist config. You should either delete the first `<VirtualHost >` from this instruction or change "Paste the following snippet into the file" to "Append the following to the `spacialist-lumen.conf` file".